src/components/one-time-donation/steps/Success.tsx

```diff
@@ -19,11 +21,13 @@ export default function Success({ donationId }: { donationId?: string }) {
       <Grid container rowSpacing={2} justifyContent={'center'} textAlign="center">
         <Grid item xs={12}>
           <Typography variant="h4" fontSize={14}>
-            {t('success.title')}
+            {(payment && t('success.title-bank')) || t('success.title')}
```

Comment:
If you need to distinguish if payment was by bank, please check if you still have access to form data and use the payment type from there paymentField.value === 'bank'

Reply:
> If you need to distinguish if payment was by bank, please check if you still have access to form data and use the payment type from there paymentField.value === 'bank'

Yep, we can access the field 
![image](https://user-images.githubusercontent.com/67734870/181795472-16407b58-a299-4432-91b1-4e3a2c045ad2.png)
so i will do it on this way....